prov/rxm/src/rxm.h

```diff
@@ -250,6 +251,18 @@ struct rxm_tx_buf {
 	struct rxm_pkt pkt;
 };
 
+struct rxm_rma_buf {
+	/* Must stay at top */
+	struct rxm_buf hdr;
+
+	/* Used for non-inject transfers */
+	struct fi_msg_rma msg;
+	struct rxm_iov rxm_iov[RXM_IOV_LIMIT];
```

Comment:
Are these fields going to be used in subsequent patches? They don't seem to be getting used in this patch and the reason to have a separate rxm_rma_buf instead rxm_tx_buf is not obvious.

Reply:
Yes, these fields are utilized in the new prepared commit (2nd commit)